crucible/src/Lang/Crucible/Simulator/RecordAndReplay.hs

```diff
@@ -273,6 +284,19 @@ getRecordedTrace globals (RecordState g) sym evalBool = do
         Just (W4.UnicodeLiteral s') -> pure s'
         Nothing -> panic "getRecordedTrace" ["Non-literal trace element?"]
 
+{- | Inserts a recorded trace into the state's replay trace variable
+The replay feature will follow this trace if it is enabled
+-}
+insertReplayTrace ::
+  (HasReplayState p p sym ext rtp) =>
+  C.SimState p sym ext rtp f args ->
+  C.RegValue sym TraceType ->
+  C.SimState p sym ext rtp f args
+insertReplayTrace st v = do
+  let simCtx = st ^. C.stateContext
+  let g = simCtx ^. C.cruciblePersonality . replayState . traceGlobal
+  st & C.stateGlobals %~ C.insertGlobal g v
```

Comment:
Now that this is no longer exported it also is basically noise in the diff (see above comment).